### PR TITLE
Give users option of overwriting

### PR DESCRIPTION
**Pull request by DanRoscigno**
Some people do not like the way the `curl | sh` download increments the filename (for example, if `clickhouse` exists in the current dir, the script will download `clickhouse.1` etc.)

This PR adds a prompt:
```
ClickHouse binary clickhouse already exists. Overwrite? [y/N] 
```
If the user responds `y` or `Y`, then the file `clickhouse` is removed and the script continues and downloads to `clickhouse`.  If the user responds with anything else, then the normal iteration over filenames occurs and the first unused filename is used for the download.  I only tested on Linux as I do not have a macOS machine.

This PR also removes the instruction on Linux to install as there is then no way to manage upgrades.

### Changelog category (leave one):
- Build/Testing/Packaging Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Adds a prompt to allow the removal of an existing `cickhouse` download when using "curl | sh" download of ClickHouse. Prompt is "ClickHouse binary clickhouse already exists. Overwrite? [y/N] " 

### Documentation entry for user-facing changes

- [x] Documentation is written (mandatory for new features)
(self documenting)

### CODE REVIEW

**Comment by alexey-milovidov** [comment]
It also requires https://github.com/ClickHouse/ClickHouse/issues/46854

**Comment by DanRoscigno** [comment]
related to https://github.com/ClickHouse/ClickHouse/issues/46808

File: docs/_includes/install/universal.sh [hunk 1]
```diff
@@ -76,9 +85,9 @@ echo
...
-    echo
-    echo "You can also install it:
-    sudo ./${clickhouse} install"
-fi
+#if [ "${OS}" = "Linux" ]
+#then
+    #echo
+    #echo "You can also install it:
+    #sudo ./${clickhouse} install"
+#fi
```
Comment (yakov-olkhovskiy) [inline docs/_includes/install/universal.sh:93]: if we don't need this it can be just removed

**Review by yakov-olkhovskiy — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
> This PR also removes the instruction on Linux to install as there is then no way to manage upgrades.

I'm using it every time to install ClickHouse.

**Comment by yakov-olkhovskiy** [comment]
Then probably makes sense to add a note that this method of installation is not recommended and refer to relevant docs...

**Pull request merged**